config/core/test-configs.yaml

```diff
@@ -1752,6 +1759,15 @@ test_configs:
       - kselftest-seccomp
       - ltp-fcntl-locktests
       - ltp-pty
+
+  - device_type: asus-C436FA-Flip-hatch
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
+      - ltp-ipc
+      - ltp-mm
       - ltp-timers
```

Comment:
It looks like `ltp-timers` is being moved from rammus to hatch, but both need to run it:
https://kernelci.org/docs/tests/ltp/

Reply:
tested OK on staging: https://staging.kernelci.org/test/plan/id/6106d025980cdbf00ed74fa4/

Reply:
Right, but rammus is missing now because of the issue mentioned in my previous comment:
https://staging.kernelci.org/test/job/kernelci/branch/staging-next/kernel/staging-next-20210801.0/plan/ltp-timers/

Reply:
OK I think it's fixed in your last version, let's wait for the next staging results to confirm.  If it all goes well, we can merge it tonight or tomorrow.

Reply:
Results are back on rammus:
https://staging.kernelci.org/test/plan/id/610827a0b17c38125cd74f7a/

And also on hatch:
https://staging.kernelci.org/test/plan/id/610826b133392f5540d74f7a/